stdpopsim/catalog/arabidopsis_thaliana.py

```diff
@@ -81,95 +81,96 @@
 
 
 # FIXME this documentation needs to be filled out.
-class _Durvasula2017MSMC(stdpopsim.Model):
-    id = "SMA_1pop"
-    name = "South Middle Atlas population size history"
-    description = """
-        Model estimated from two homozygous individuals from the South Middle Atlas
-        using MSMC (TODO: more detail).
-    """
+def _Durvasula2017MSMC():
```

Comment:
See earlier point about function names. Maybe change this to "_sma_1pop" (i.e., the ID of the model with a leading underscore)?